beacon-chain/casper/incentives.go

```diff
@@ -76,13 +76,6 @@ func RewardQuotient(validators []*pb.ValidatorRecord) uint64 {
 	return params.GetConfig().BaseRewardQuotient * mathutil.IntegerSquareRoot(totalDepositETH)
 }
 
-// SlotMaxInterestRate returns the interest rate for a validator in a slot, the interest
-// rate is targeted for a compunded annual rate of 3.88%.
-func SlotMaxInterestRate(validators []*pb.ValidatorRecord) float64 {
```

Comment:
This is an unused method that exists in the spec for demonstration purpose only.